ci/audit.yml

```diff
@@ -5,8 +5,8 @@ platform: linux
 image_resource:
   type: docker-image
   source:
-    repository: node
-    tag: 14.15.4
```

Comment:
general question is not the official node repo not ok ?

Reply:
🤷 I just used what the Florence build files use.

Reply:
So I was wondering why we need to move to that image ?

Reply:
Not sure what you are asking.

I just remember in stand up I mentioned that this library was having trouble passing it's audit checks because it uses a outdated version of node, you replied that Jon upgraded Florence to use a later version of node so I just copied what Florence uses in these files, the only difference is they use the latest tag whilst I specify a version which was a request by Linden.

Reply:
Making a new branch using the node repo now